enterprise/cmd/repo-updater/authz/main_test.go

```diff
@@ -5,6 +5,8 @@ import (
 	"os"
 	"testing"
 
+	secretsPkg "github.com/sourcegraph/sourcegraph/internal/secrets"
+
 	"github.com/inconshreveable/log15"
```

Comment:
nit:
```suggestion
	"github.com/inconshreveable/log15"

	secretsPkg "github.com/sourcegraph/sourcegraph/internal/secrets"
```

Same in other `main_test.go` files.